### ISSUE TITLE
A-TET-001004 Tetra PCB Assembly 5V Regulator Issue

### ISSUE THREAD

**Issue by mechatroniks-git** [tetrabiodistributed/project-tetra#112]
Board serial # 00 is not outputting 5VDC

**Comment by mechatroniks-git** [comment]
https://github.com/tetrabiodistributed/project-tetra/tree/master/02_Development/FlowMeter/RPi%20Board

Datasheet: http://www.xlsemi.com/datasheet/XL4015%20datasheet.pdf

**Comment by hydronics2** [comment]
@mechatroniks-git I just reviewed the pinout and reference schematic. The pinout is correct. All the parts seem placed as shown. I am missing this cap on the FB line. (33nf)
![image](https://user-images.githubusercontent.com/3589948/89218046-2d423580-d582-11ea-8782-f529c7ebe731.png)

**Comment by hydronics2** [comment]
I figured it out I think... they installed the diode in backwards... The orientation was correct when I reviewed it online but something must have happened at the boardshop...!  I'll get it swapped around and see what happens... 
![image](https://user-images.githubusercontent.com/3589948/89222596-291a1600-d58a-11ea-980b-458f47cebb68.png)

**Comment by hydronics2** [comment]
Ok... we got 5V! woohoo...  I turned the diode around and added a 33nF cap along side the 10K resistor (not sure if the cap is 100% necessary but following the data sheet)
![image](https://user-images.githubusercontent.com/3589948/89223588-e5281080-d58b-11ea-98f9-1c28c1b8d2f6.png)

notice I made the jumper... so now you have 5V LED lit and the RPI 5V LED lit.... + the 12V LED lit...so that's 3 indicator LEDs (all accept the 3V rail which comes from the PI).   Pulling 17ma at 12V. 

![image](https://user-images.githubusercontent.com/3589948/89223732-291b1580-d58c-11ea-8d6e-3300aa7a4db8.png)
